### ISSUE TITLE
Coppermine shows critical database error when clicking on thumbnail within album to view the full-sized picture

### ISSUE THREAD

**Issue by overtesters** [coppermine-gallery/cpg1.6.x#75]
Hi. First of all, my apologies if I did something wrong. My first post, and I can't register on coppermine's forum to mention my issue there.

My Coppermine Installation is quite old (first use was in December 2006), but I always kept it up to date. Now it's running Release 1.6.26.

With CPG Release 1.6.25, I noticed occasionally a critical database error when me and my visitors clicked on a thumbnail in an album to show the full sized picture. After testing and checking debug messages, I found out this happened only when my gallery was accessed by IPv6-Addresses.

Long story short, In my database cpg1410_hit_stats, row ip had a value 20 which of course isn't long enough for an IPv6-Address. I changed that value to 40. The problem has been solved since then. :)

I just wanted to place this here, no idea if this Issue has been fixed. Maybe this step has been overlooked during my update process because of the age of my installation.

**Comment by ron4mac** [comment]
Thank you for your effort and feedback. The necessary changes have been made to correct the issue in the next release version (1.6.27)
